templates/core/index.html

```diff
@@ -1,817 +1,753 @@
+        videoLink.addEventListener('click', function(e) {
+          e.preventDefault();
+          videoContainer.classList.remove('u-hide');
+          videoLink.classList.add('u-hide');
+        });
+      });
+    </script>
   </div>
 </section>
-<section class="p-strip">
-  <div class="u-fixed-width">
-    <h2>Why choose Ubuntu Core?</h2>
-  </div>
-  <div class="row" style="padding-top: 60px; padding-bottom: 60px;">
-    <div class="col-3">
-      <h3>Security First</h3>
-    </div>
-    <div class="col-9">
-      <p class="p-heading--5">Imagine a world where every device is trustworthy. We designed every aspect of Ubuntu Core to create the most secure embedded Linux ever, with a 10y LTS commitment.</p>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-3">
-      <div class="p-heading-icon">
-        <div class="p-heading-icon__header is-stacked">
-          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/a0f23d1b-Savings.svg" alt="">
-          <h4 class="p-heading-icon__title p-heading--5">Reduce the cost of Linux</h4>
+
+<section class="p-section">
+  <div class="row--50-50 p-section--shallow">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>An OS with all the infrastructure</h2>
+    </div>
+    <div class="col">
+      <p>Ubuntu Core is more than just your embedded Linux OS, it's your whole deployment infrastructure. Build targeted
+        production-ready images, unlock over-the air (OTA) updates, deploy in air gap environments, manage your fleet of
+        devices, achieve real-time operations, and more.</p>
+      <p>Ubuntu Core will transform the way you deploy devices with the longest support window in the industry.</p>
+    </div>
+  </div>
+  <div class="p-equal-height-row has-3rd-divider">
+    <div class="p-equal-height-row__col">
+      <div class="p-equal-height-row__item u-hide--small">
+        <div class="p-media-container">
+          <img width="320" src="https://assets.ubuntu.com/v1/834e417a-continuous-security.jpg" alt="" />
         </div>
-        <p>Low-level devices development is complex and resource-intensive. With Ubuntu Core and Canonical services, you can accelerate the time-to-market and reduce your costs.</p>
       </div>
-    </div>
-    <div class="col-3">
-      <div class="p-heading-icon">
-        <div class="p-heading-icon__header is-stacked">
-          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/da8d1235-ubuntu-cof.svg" alt="">
-          <h4 class="p-heading-icon__title p-heading--5">Loved by developers</h4>
-        </div>
-        <p>Core is Ubuntu; open source, binary-compatible and backed by a strong developer community.
-          Improve productivity with a unified and application-centric developer experience.</p>
+      <div class="p-equal-height-row__item">
+        <hr class="p-rule--highlight u-hide--medium u-hide--small is-muted" />
+        <h5 class="p-heading--5">Continuous security</h5>
```

Comment:
```suggestion
        <h3 class="p-heading--5">Continuous security</h3>
```
Please adjust for all of these so we're not jumping from h2 to h5